content/03-types/03-document/01-what-are-document-dbs.mdx

```diff
@@ -0,0 +1,107 @@
+
+Document databases can avoid many of these shortcomings experienced by relational databases due to how their systems structure data. The coordination between different hosts can be minimized by collocating related data together in a single document. [Sharding](/intro/database-glossary#shard) datasets is a much more common strategy in document databases. This is because document-based operations typically don't require much coordination since many actions target individual records.
+
+Because fewer [constraints](/intro/database-glossary#constraint) and links exist between individual documents and collections within document databases, coordination is often more accessible, and operations tend to be more self-contained. This allows document database providers to prioritize performance and [availability](/intro/database-glossary#availability), where relational databases force concessions in the name of [consistency](/intro/database-glossary#consistency). 
+
+This results in a trade-off implicating the safety of your data and how well your systems can handle outages and network partitions. The significant difference is that document databases tend to have much more flexibility in tuning the level of consistency versus performance and availability. In contrast, relational databases often require consistency always to be the priority.
+
+### Agility 
+The document model’s [schema-less](/intro/database-glossary#schema) capabilities can have databases up and running very quickly. Minimal maintenance is required once you create a document, and you can start inserting objects as documents immediately. 
+
+The agility provided by document databases makes it, so you do not have to know the exact structure of your data at the time of implementation. Data models are subject to change, and formulating a clear plan can be challenging when development begins. The combination of agility and flexibility allows developers to spin up a database instance and populate it with collections of documents right away and evolve the model alongside the application's evolution.
+
+However, with this lack of schema comes the trade-off. The consistency of your data needs to be managed continuously rather than from the plan of a pre-defined schema. There is an advantage to having a good picture of your data's look and access patterns ahead of time. Relational databases force this consideration.  
+
+## Conclusion
+This article covered document databases and why they are one of the most popular NoSQL offerings. We covered the structure and capabilities of documents and the document model advantages, and their associated trade-offs.
+
+Document databases offer a different approach than relational databases to organizing and accessing data. This evolution from only the traditional models is exciting for the decision it now gives to developers. Based on your application, you can decide which features and advantages best align with your philosophies and goals. 
```

Comment:
```suggestion
Document databases offer a different approach than relational databases to organizing and accessing data. This evolution from only the traditional models is exciting for the choice it now gives to developers. Based on your application, you can decide which features and advantages best align with your philosophies and goals. 
```
Maybe? Just sounds more of a benefit rather than a responsibility to me lol

Reply:
I like that